mlflow/store/dbmodels/models.py

```diff
@@ -60,6 +60,8 @@ def _create_entity(base, model):
                     obj = SourceType.from_string(obj)
                 elif k == "status":
                     obj = RunStatus.from_string(obj)
+                elif k == "experiment_id":
```

Comment:
Are only runs created in sql alchemy? If an experiment is returned would it be deserialized with to_mlflow_entity()?

Reply:
nvm. The experiment_id cast was added for SqlExperiment below